test/debugger/test-debugger-stop.js

```
@@ -0,0 +1,24 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+var debug = require('_debugger');
+
+var fork = require('child_process').fork;
+
+var scriptToDebug = common.fixturesDir + '/debug-connect.js';
+
+var nodeProcess;
+
+try {
+  process._debugProcess(process.pid);
+} catch(ex) {
+  assert.fail('Debugging should not fail');
```

I'd remove the try/catch and simply let the exception bubble up.